website/src/main/webapp/prototype_script.js

```diff
@@ -171,6 +171,59 @@ function openSearch() {
 function closeSearch() {
   document.getElementById("myOverlay").style.display = "none";
 }
+/**
+ * Checks if current user is admin
+ * Redirects invalid users to home page
+ */
+
+function loadAdminPage() {
+  const qs = "/admin";
+  fetch(qs).then(response => response.json()).then(adminText => {
+    if (adminText['user'] === 'Admin') {
+      setUpForAdmin(adminText);
+    } else { 
+      alert("You are unauthorized for this page");
+      window.location.replace("/index.html");
+    }
+  });
+}
+
+/**
+ * Sets up UI on page for admin
+ * Adds functionality for dynamically selecting action items
+ */
+
+function setUpForAdmin(adminText) {
+  //admin UI to page
+      document.getElementById('navActionItems').style.display = 'block';
+      document.getElementById('uploadOrgs').innerHTML += adminText['uploadCSV'];
```

Comment:
optional: I'm just curious why you chose to += append to innerHTML rather than assigning with = when supposedly there is nothing there previously